src/libraries/Common/tests/System/Security/Cryptography/MLKemCngTests.Windows.cs

```diff
@@ -0,0 +1,201 @@
+        }
+
+        [Fact]
+        public static void MLKemCng_NonExportable_ExportEncapsulationKeyAlwaysWorks()
+        {
+            using CngKey key = MLKemCngTests.ImportPrivateSeed(
+                MLKemAlgorithm.MLKem512,
+                MLKemTestData.MLKem512PrivateSeed,
+                CngExportPolicies.None);
+
+            using MLKemCng kem = new MLKemCng(key);
+            byte[] exportedKey = kem.ExportEncapsulationKey();
+            AssertExtensions.SequenceEqual(MLKemTestData.MLKem512EncapsulationKey, exportedKey);
+        }
+    }
+
+    public abstract class MLKemCngTests : MLKemBaseTests
+    {
+        protected abstract CngExportPolicies ExportPolicies { get; }
+
+        public override MLKem GenerateKey(MLKemAlgorithm algorithm)
+        {
+            using (CngKey key = GenerateCngKey(algorithm, ExportPolicies))
+            {
+                return new MLKemCng(key);
+            }
+        }
+
+        public override MLKem ImportPrivateSeed(MLKemAlgorithm algorithm, ReadOnlySpan<byte> seed)
+        {
+            using (CngKey key = ImportPrivateSeed(algorithm, seed, ExportPolicies))
+            {
+                return new MLKemCng(key);
+            }
+        }
+
+        public override MLKem ImportDecapsulationKey(MLKemAlgorithm algorithm, ReadOnlySpan<byte> source)
+        {
+            using (CngKey key = ImportMLKemKey(KeyBlobMagicNumber.BCRYPT_MLKEM_PRIVATE_MAGIC, algorithm, source, ExportPolicies))
+            {
+                return new MLKemCng(key);
+            }
+        }
+
+        public override MLKem ImportEncapsulationKey(MLKemAlgorithm algorithm, ReadOnlySpan<byte> source)
+        {
+            using (CngKey key = ImportMLKemKey(KeyBlobMagicNumber.BCRYPT_MLKEM_PUBLIC_MAGIC, algorithm, source, ExportPolicies))
+            {
+                return new MLKemCng(key);
+            }
+        }
+
+        internal static CngKey GenerateCngKey(MLKemAlgorithm algorithm, CngExportPolicies exportPolicies)
+        {
+            CngProperty parameterSet = GetCngProperty(algorithm);
+
+            CngKeyCreationParameters creationParameters = new();
+            creationParameters.Parameters.Add(parameterSet);
+            creationParameters.ExportPolicy = exportPolicies;
+
+            return CngKey.Create(CngAlgorithm.MLKem, keyName: null, creationParameters);
+        }
+
+        internal static CngKey ImportPrivateSeed(
+            MLKemAlgorithm algorithm,
+            ReadOnlySpan<byte> seed,
+            CngExportPolicies exportPolicies)
+        {
+            return ImportMLKemKey(KeyBlobMagicNumber.BCRYPT_MLKEM_PRIVATE_SEED_MAGIC, algorithm, seed, exportPolicies);
+        }
+
+        [Fact]
+        public void MLKemCng_Ctor_ArgValidation()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("key", static () => new MLKemCng(null));
+        }
+
+        [Fact]
+        public void MLKemCng_Ctor_KeyWrongAlgorithm()
```

Comment:
Does xunit allow mixing static and instance on the same type? Because we don't really need this test (or the no-null test) to run for all 3 export policy modes.

It's a trivial thing, so if it isn't easy to fix it isn't worth fixing... but thought I'd ask.